apps/designer/app/designer/features/props-panel/props-panel.tsx

```diff
@@ -217,6 +251,8 @@ export const PropsPanel = ({
     </Button>
   );
 
+  const existingProps = userProps.map((userProp) => userProp.prop);
```

Comment:
its not the initialProps?

Reply:
Yes it's all props the user has on the screen. To avoid having 2 of the same props on screen.